### PR TITLE
Add restorePointSetting to cloudPcUserSetting

### PR DESCRIPTION
**Pull request by RuiHou105**

### CODE REVIEW

**Comment by RuiHou105** [comment]
**Roadmap workitem:** 

Feature 12481: [Devices and apps][Cloud PC] Add restorePointSetting to cloudPcUserSetting

**Review by AshleyYangSZ — APPROVED** [review]

**Comment by RuiHou105** [comment]
@isvargasmsft  @Danielabom  @JarbasHorst  @Lauragra  
As the graph schema already went alive, could you please help review this PR? Thanks!

File: api-reference/beta/resources/cloudpcrestorepointsetting.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents specific settings of point-in-time restore.
+
+## Properties
+
+|Property|Type|Description|
+|:---|:---|:---|
+|frequencyInHours|Int32|The time interval for auto taking snapshots (restore points) of a Cloud PC. Possible values are: 4 hours, 6 hours, 12 hours, 18 hours, 24 hours. The default frequency is 12 hours.|
```
Comment (Danielabom) [inline api-reference/beta/resources/cloudpcrestorepointsetting.md:22]: Line 22: The second part of the description lists the possible values of the property. However, the type is an Int32 which allows the user to enter any number that qualifies as an Int32. How do you control that users will not enter other values from the ones listed in this description?

In addition, the get and update examples display a value of 16 hours which is not even listed as a possible value here.

File: changelog/Microsoft.CloudManagedDesktop.json [hunk 2]
```diff
@@ -1,5 +1,31 @@
...
+                    "Target": "cloudPcRestorePointSetting"
+                }
+            ],
+            "Id": "fc913e87-36e8-400f-ac60-05ae8f512440",
+            "Cloud": "prd",
+            "Version": "beta",
+            "CreatedDateTime": "2022-01-06T14:51:08.271Z",
+            "WorkloadArea": "Devices and apps",
+            "SubArea": "Cloud PC"
+        },
```
Comment (Danielabom) [inline changelog/Microsoft.CloudManagedDesktop.json:28]: Missing in this PR:
Entity: cloudPcSnapshot
Complex type: cloudPcSnapshotstatus
Complex type: cloudPcBulkRemoteActionResult
Enum: restoreTimeRange

Please review whether these resources have to be added.

**Review by Danielabom — CHANGES_REQUESTED** [review]
Hi Rui, Thank you for working on this PR. Please review my comments and address them accordingly. 
Thanks,
Dani

File: api-reference/beta/resources/cloudpcrestorepointsetting.md (hunk 1 above)
Comment (RuiHou105) [reply api-reference/beta/resources/cloudpcrestorepointsetting.md:22]: Hi @Danielabom 
Q1: Here I attach the UX screenshot. You can see that there is a dropdown with options that users can choose. So it's impossible that users will enter an invalid number. 
image


Q2: All possible values are: 4, 6, 12, 16, 24. I corrected the values. Please check. Thanks!

File: changelog/Microsoft.CloudManagedDesktop.json (hunk 2 above)
Comment (RuiHou105) [reply changelog/Microsoft.CloudManagedDesktop.json:133]: Hi @Danielabom ,
Entity: `cloudPcSnapshot` and complex type: `cloudPcSnapshotStatus` are recorded in this PR: https://github.com/microsoftgraph/microsoft-graph-docs/pull/15416

complex type: `cloudPcBulkRemoteActionResult` and enumeration `restoreTimeRange` are recorded  in this one: 
https://github.com/microsoftgraph/microsoft-graph-docs/pull/15556

File: api-reference/beta/resources/cloudpcrestorepointsetting.md (hunk 1 above)
Comment (Danielabom) [reply api-reference/beta/resources/cloudpcrestorepointsetting.md]: Line 22: The first part of my comment was not addressed. Please consider changing line 22 to: "The time interval in hours to take snapshots automatically (restore points) of a Cloud PC. Possible values are 4, 6, 12 16, 18, and 24. The default frequency is 12 hours."

Comment (RuiHou105) [reply api-reference/beta/resources/cloudpcrestorepointsetting.md]: Thanks @Danielabom . Description updated. Please check.

**Review by Danielabom — APPROVED** [review]

**Comment by RuiHou105** [comment]
@Lauragra , if you don't have other concerns. could you please help merge this PR? Thanks!

**Review by Lauragra — APPROVED** [review]

**Pull request merged**